scripts/manage_chalice_deployed.py

```diff
@@ -19,6 +19,8 @@
 
 s3 = boto3.client('s3')
 
+# TODO: Delete this whole file. But first, all deployments should be configured
```

Comment:
It can be deleted right away, but it would make it harder to switch over a deployment. You would have to run `make -C lambdas delete` before this branch was merged, and then run `make deploy` followed by `make terraform` afterwards in order to update how your deployment is done.

Leaving these scripts in allow us to remove the lambda resources using the old mechanism (with Chalice) after this PR is merged.

Reply:
As per our discussion, I'll delete these in this PR.